2025-05-16 09:47:58,900 - INFO - URL OK: http://hls1.sua.tv/live/globotvbahiafhdbr2/s.m3u8
2025-05-16 09:48:47,261 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112660: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out.
2025-05-16 09:49:44,116 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112664: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out.
2025-05-16 09:50:01,661 - INFO - URL OK: http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112665
2025-05-16 09:50:17,300 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112666: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out. (read timeout=15)
2025-05-16 09:50:35,635 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112667: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out.
2025-05-16 09:50:51,267 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112668: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out. (read timeout=15)
2025-05-16 09:51:06,735 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112669: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out. (read timeout=15)
2025-05-16 09:51:24,047 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112670: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out.
2025-05-16 09:51:37,044 - INFO - URL OK: http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112671
2025-05-16 09:51:52,516 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112672: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out. (read timeout=15)
2025-05-16 09:52:09,991 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112673: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out.
2025-05-16 09:52:25,458 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112674: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out. (read timeout=15)
2025-05-16 09:52:40,927 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112675: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out. (read timeout=15)
2025-05-16 09:52:57,641 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112676: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out.
2025-05-16 09:53:13,143 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112677: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out. (read timeout=15)
2025-05-16 09:53:29,866 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112678: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out.
2025-05-16 09:53:32,166 - INFO - URL OK: http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112679
2025-05-16 09:53:53,964 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112680: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out.
2025-05-16 09:54:10,656 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112681: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out.
2025-05-16 09:54:27,440 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112682: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out.
2025-05-16 09:54:30,216 - INFO - URL OK: http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112683
2025-05-16 09:54:52,706 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112684: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out.
2025-05-16 09:55:09,602 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112685: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out.
2025-05-16 09:55:26,501 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112686: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out.
2025-05-16 09:55:26,880 - WARNING - URL Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112687: Status Code 403
2025-05-16 09:55:44,787 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112688: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out.
2025-05-16 09:56:01,492 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112689: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out.
2025-05-16 09:56:18,688 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112690: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out.
2025-05-16 09:56:30,346 - INFO - URL OK: http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112691
2025-05-16 09:56:52,049 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112692: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out.
2025-05-16 09:57:07,518 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112693: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out. (read timeout=15)
2025-05-16 09:57:28,412 - INFO - URL OK: http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112694
2025-05-16 09:57:28,433 - ERROR - Request Error http://15.235.15.230:14783/: HTTPConnectionPool(host='15.235.15.230', port=14783): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa09e309430>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-16 09:57:28,453 - ERROR - Request Error http://15.235.15.230:14478/: HTTPConnectionPool(host='15.235.15.230', port=14478): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa09e3094c0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-16 09:57:28,475 - ERROR - Request Error http://15.235.15.230:14492/: HTTPConnectionPool(host='15.235.15.230', port=14492): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa09e3082f0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-16 09:57:28,496 - ERROR - Request Error http://15.235.15.230:14485/: HTTPConnectionPool(host='15.235.15.230', port=14485): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa09e2cbdd0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-16 09:57:28,517 - ERROR - Request Error http://15.235.15.230:14495/: HTTPConnectionPool(host='15.235.15.230', port=14495): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa09e2c9760>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-16 09:57:28,537 - ERROR - Request Error http://15.235.15.230:14502/: HTTPConnectionPool(host='15.235.15.230', port=14502): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa09e2cb3b0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-16 09:57:28,558 - ERROR - Request Error http://15.235.15.230:14506/: HTTPConnectionPool(host='15.235.15.230', port=14506): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa09e2ca060>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-16 09:57:28,579 - ERROR - Request Error http://15.235.15.230:14504/: HTTPConnectionPool(host='15.235.15.230', port=14504): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa09e309700>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-16 09:57:28,600 - ERROR - Request Error http://15.235.15.230:14508/: HTTPConnectionPool(host='15.235.15.230', port=14508): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa09e3089b0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-16 09:57:28,620 - ERROR - Request Error http://15.235.15.230:14513/: HTTPConnectionPool(host='15.235.15.230', port=14513): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa09e2c90a0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-16 09:57:28,641 - ERROR - Request Error http://15.235.15.230:14512/: HTTPConnectionPool(host='15.235.15.230', port=14512): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa09e2cad50>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-16 09:57:28,662 - ERROR - Request Error http://15.235.15.230:14510/: HTTPConnectionPool(host='15.235.15.230', port=14510): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa09e2cbf20>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-16 09:57:28,682 - ERROR - Request Error http://15.235.15.230:14516/: HTTPConnectionPool(host='15.235.15.230', port=14516): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa09e2c9b80>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-16 09:57:28,703 - ERROR - Request Error http://15.235.15.230:14492/AFDTV: HTTPConnectionPool(host='15.235.15.230', port=14492): Max retries exceeded with url: /AFDTV (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa09e308e30>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-16 09:57:28,724 - ERROR - Request Error http://15.235.15.230:14487/AFDTV: HTTPConnectionPool(host='15.235.15.230', port=14487): Max retries exceeded with url: /AFDTV (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa09e30a4b0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-16 09:57:28,744 - ERROR - Request Error http://15.235.15.230:14499/AFDTV: HTTPConnectionPool(host='15.235.15.230', port=14499): Max retries exceeded with url: /AFDTV (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa09e2c8ad0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-16 09:57:28,765 - ERROR - Request Error http://15.235.15.230:14513/AFDTV: HTTPConnectionPool(host='15.235.15.230', port=14513): Max retries exceeded with url: /AFDTV (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa09e2c94f0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-16 09:57:28,786 - ERROR - Request Error http://15.235.15.230:14513/: HTTPConnectionPool(host='15.235.15.230', port=14513): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa09e2c8ef0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-16 09:57:29,078 - INFO - URL OK: http://hls1.sua.tv/live/globotvbahiafhdbr2/s.m3u8
2025-05-16 09:57:29,353 - WARNING - URL Error http://s1.1-cup.net:2095/solucao111hgfgh77hu78/xplus1111XXhjgyu87/51: Status Code 404
2025-05-16 09:57:29,374 - ERROR - Request Error http://15.235.15.230:14487/: HTTPConnectionPool(host='15.235.15.230', port=14487): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa09e2ca120>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-16 09:57:29,969 - WARNING - URL Error https://cdn-3.nxplay.com.br/GLOBO_SP_TK/tracks-v1a1/mono.m3u8: Status Code 403
2025-05-16 09:57:30,238 - INFO - URL OK: http://hls1.sua.tv/live/globotvbahiafhdbr2/s.m3u8
2025-05-16 09:57:30,531 - INFO - URL OK: http://hls1.sua.tv:80/live/globosphd/s.m3u8
2025-05-16 09:57:31,711 - INFO - URL OK: http://metv.fun:80/4x6cXF/JDqRqj/90059.m3u8
2025-05-16 09:57:32,382 - WARNING - URL Error http://metv.fun:80/4x6cXF/JDqRqj/91431.m3u8: Status Code 404
2025-05-16 09:57:33,463 - INFO - URL OK: http://lidns.cc:80/4x6cXF/JDqRqj/90034.m3u8
2025-05-16 09:57:34,350 - INFO - URL OK: http://lidns.cc:80/4x6cXF/JDqRqj/91421.m3u8
2025-05-16 09:57:35,370 - INFO - URL OK: http://lidns.cc:80/4x6cXF/JDqRqj/91435.m3u8
2025-05-16 09:57:36,258 - INFO - URL OK: http://lidns.cc:80/4x6cXF/JDqRqj/91463.m3u8
2025-05-16 09:57:37,248 - INFO - URL OK: http://metv.fun:80/4x6cXF/JDqRqj/90064.m3u8
2025-05-16 09:57:38,251 - INFO - URL OK: http://lidns.cc:80/4x6cXF/JDqRqj/90019.m3u8
2025-05-16 09:57:39,164 - INFO - URL OK: http://lidns.cc:80/4x6cXF/JDqRqj/95441.m3u8
2025-05-16 09:57:40,170 - INFO - URL OK: http://lidns.cc:80/4x6cXF/JDqRqj/91405.m3u8
2025-05-16 09:57:41,047 - INFO - URL OK: http://lidns.cc:80/4x6cXF/JDqRqj/91413.m3u8
2025-05-16 09:57:41,948 - INFO - URL OK: http://lidns.cc:80/4x6cXF/JDqRqj/120082.m3u8
2025-05-16 09:57:42,942 - INFO - URL OK: http://lidns.cc:80/4x6cXF/JDqRqj/91443.m3u8
2025-05-16 09:57:43,831 - INFO - URL OK: http://lidns.cc:80/4x6cXF/JDqRqj/91445.m3u8
2025-05-16 09:57:44,823 - INFO - URL OK: http://lidns.cc:80/4x6cXF/JDqRqj/95483.m3u8
2025-05-16 09:57:45,829 - INFO - URL OK: http://lidns.cc:80/4x6cXF/JDqRqj/97396.m3u8
2025-05-16 09:57:46,824 - INFO - URL OK: http://lidns.cc:80/4x6cXF/JDqRqj/91456.m3u8
2025-05-16 09:57:47,443 - WARNING - URL Error http://lidns.cc:80/4x6cXF/JDqRqj/91460.m3u8: Status Code 404
2025-05-16 09:57:48,467 - INFO - URL OK: http://lidns.cc:80/4x6cXF/JDqRqj/95448.m3u8
2025-05-16 09:57:49,464 - INFO - URL OK: http://lidns.cc:80/4x6cXF/JDqRqj/91461.m3u8
2025-05-16 09:57:50,352 - INFO - URL OK: http://lidns.cc:80/4x6cXF/JDqRqj/91467.m3u8
2025-05-16 09:57:51,232 - INFO - URL OK: http://lidns.cc:80/4x6cXF/JDqRqj/91470.m3u8
2025-05-16 09:57:52,111 - INFO - URL OK: http://lidns.cc:80/4x6cXF/JDqRqj/91471.m3u8
2025-05-16 09:57:53,137 - INFO - URL OK: http://lidns.cc:80/4x6cXF/JDqRqj/91473.m3u8
2025-05-16 09:57:54,014 - INFO - URL OK: http://lidns.cc:80/4x6cXF/JDqRqj/91477.m3u8
2025-05-16 09:57:55,016 - INFO - URL OK: http://lidns.cc:80/4x6cXF/JDqRqj/91479.m3u8
2025-05-16 09:57:55,036 - ERROR - Request Error http://15.235.15.230:14783/: HTTPConnectionPool(host='15.235.15.230', port=14783): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa09e2c9100>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-16 09:57:55,057 - ERROR - Request Error http://15.235.15.230:14478/: HTTPConnectionPool(host='15.235.15.230', port=14478): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa09e446510>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-16 09:57:55,078 - ERROR - Request Error http://15.235.15.230:14492/: HTTPConnectionPool(host='15.235.15.230', port=14492): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa09e2cbc20>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-16 09:57:55,099 - ERROR - Request Error http://15.235.15.230:14485/: HTTPConnectionPool(host='15.235.15.230', port=14485): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa09e2c9c10>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-16 09:57:55,119 - ERROR - Request Error http://15.235.15.230:14495/: HTTPConnectionPool(host='15.235.15.230', port=14495): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa09e2c9400>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-16 09:57:55,140 - ERROR - Request Error http://15.235.15.230:14502/: HTTPConnectionPool(host='15.235.15.230', port=14502): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa09e2c8ad0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-16 09:57:55,161 - ERROR - Request Error http://15.235.15.230:14506/: HTTPConnectionPool(host='15.235.15.230', port=14506): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa09e3083b0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-16 09:57:55,182 - ERROR - Request Error http://15.235.15.230:14504/: HTTPConnectionPool(host='15.235.15.230', port=14504): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa09e30ac60>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-16 09:57:55,203 - ERROR - Request Error http://15.235.15.230:14508/: HTTPConnectionPool(host='15.235.15.230', port=14508): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa09e2c9850>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-16 09:57:55,224 - ERROR - Request Error http://15.235.15.230:14513/: HTTPConnectionPool(host='15.235.15.230', port=14513): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa09e2cb200>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-16 09:57:55,244 - ERROR - Request Error http://15.235.15.230:14512/: HTTPConnectionPool(host='15.235.15.230', port=14512): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa09e2ca8d0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-16 09:57:55,265 - ERROR - Request Error http://15.235.15.230:14510/: HTTPConnectionPool(host='15.235.15.230', port=14510): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa09e2cb4d0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-16 09:57:55,286 - ERROR - Request Error http://15.235.15.230:14516/: HTTPConnectionPool(host='15.235.15.230', port=14516): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa09f3b5d00>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-16 09:57:55,307 - ERROR - Request Error http://15.235.15.230:14492/AFDTV: HTTPConnectionPool(host='15.235.15.230', port=14492): Max retries exceeded with url: /AFDTV (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa09e309f70>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-16 09:57:55,327 - ERROR - Request Error http://15.235.15.230:14487/AFDTV: HTTPConnectionPool(host='15.235.15.230', port=14487): Max retries exceeded with url: /AFDTV (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa09e2cacc0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-16 09:57:55,348 - ERROR - Request Error http://15.235.15.230:14499/AFDTV: HTTPConnectionPool(host='15.235.15.230', port=14499): Max retries exceeded with url: /AFDTV (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa09e2c91f0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-16 09:57:55,369 - ERROR - Request Error http://15.235.15.230:14513/AFDTV: HTTPConnectionPool(host='15.235.15.230', port=14513): Max retries exceeded with url: /AFDTV (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa09e2c8ec0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-16 09:57:55,389 - ERROR - Request Error http://15.235.15.230:14513/: HTTPConnectionPool(host='15.235.15.230', port=14513): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa09e2c9490>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-16 09:57:55,688 - INFO - URL OK: http://hls1.sua.tv/live/globotvbahiafhdbr2/s.m3u8
2025-05-16 09:57:55,962 - WARNING - URL Error http://s1.1-cup.net:2095/solucao111hgfgh77hu78/xplus1111XXhjgyu87/51: Status Code 404
